tests/std/tests/Dev11_0836436_get_time/test.cpp

```diff
@@ -944,3 +909,68 @@ void test_gh_4820() {
         assert(wss.str() == L"\x043a%\x043e%\x0448%E\x043a%O\x0430");
     }
 }
+
+void test_gh_4882() {
+    // GH-4882 <iomanip>: std::put_time should set failbit on invalid/out-of-range tm struct, instead of crash
+    const auto fieldValidation = [](int tm::*field, int value, const string& format) {
+        time_t t = time(nullptr);
+        tm currentTime;
+        localtime_s(&currentTime, &t);
+
+        (currentTime.*field) = value;
+
+        stringstream ss;
+        ss << put_time(&currentTime, format.c_str());
+        assert(ss.rdstate() == ios_base::goodbit);
+        const auto result = ss.str();
+        assert(result.length() == format.length() / 2);
+        assert(std::all_of(result.cbegin(), result.cend(), [](const char c) { return c == '?'; }));

```

Comment:
This test is already `using namespace std;` so there should be no `std::` qualification. (Occurs below.)